.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
I suggest that you add some html comments in the form `<!-- a normal html comment -->`, saying to fill this in and paste it into an issue on Github. Apparently html comments will not be rendered by Github.

Reply:
I'm not sure what you mean. There are html comments throughout the file. I think once we have this merged, we can add these as our templates and they will auto populate the text box after you click on <code>new issue</code>. So the information on populating the sections will be displayed as comments to the person filling out the issue and will be ignored once posted, so we will not have to read the directions for the issuer.